db/migrate/20230105155624_add_eligibility_enabled_to_countries.rb

```diff
@@ -0,0 +1,5 @@
+class AddEligibilityEnabledToCountries < ActiveRecord::Migration[7.0]
+  def change
+    add_column :countries, :eligibility_enabled, :boolean
```

Comment:
We should also add this field to the support console so someone can tick the box to enable the countries on the 1st Feb.